src/Product/Attributes/SizeType.php

```diff
@@ -67,4 +67,31 @@ public static function get_input_type(): string {
 		return SizeTypeInput::class;
 	}
 
+	/**
+	 * Returns the attribute name
+	 *
+	 * @return string
+	 */
+	public static function get_name(): string {
+		return __( 'Size Type', 'google-listings-and-ads' );
+	}
+
+	/**
+	 * Returns true if the attribute is an enum
+	 *
+	 * @return bool
+	 */
+	public static function is_enum(): bool {
+		return true;
+	}
+
+	/**
+	 * Returns the attribute sources
+	 *
+	 * @return array
+	 */
+	public static function get_sources(): array {
+		return self::get_value_options();
```

Comment:
I realize these values were already present, but I just noticed that they are different then the [help article](https://support.google.com/merchants/answer/6324497?hl=en). Do we need to double check with Google to make sure all the enums have the right set of values?

Reply:
Makes sense to double check. Shall I do that in the questions doc?

Reply:
Yes that would seem like the best place.

Reply:
Done